core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -565,4 +560,12 @@ public static CLICommand getCurrent() {
         }
     }
 
+    static public void reportException(String name, Logger logger, PrintStream stderr, Throwable e) {
```

Comment:
I would suggest to add Javadoc or to mark this API as restricted

Reply:
```suggestion
    @Restricted(NoExternalUse.class) 
    static public void reportException(String name, Logger logger, PrintStream stderr, Throwable e) {
```

Reply:
I think so jsoref, yes